core/templates/services/math-interactions.service.ts

```diff
@@ -19,6 +19,8 @@
 import { Injectable } from '@angular/core';
 import { downgradeInjectable } from '@angular/upgrade/static';
 
+import { AppConstants } from 'app.constants';
+
 const nerdamer = require('nerdamer');
```

Comment:
Please change all instances
```typescript
const nerdamer = require('nerdamer');
```
to
```typescript
import nerdamer from 'nerdamer';
```

This way the type definitions for nerdamer are inferred by the compiler.

![nerdmar1](https://user-images.githubusercontent.com/36989112/85932825-41dd3f00-b8ed-11ea-9231-823e1bb71ac4.png)
![nerdmar2](https://user-images.githubusercontent.com/36989112/85932828-44d82f80-b8ed-11ea-9697-981ebe95da1a.png)

Also I noticed that some functions are not present in their type defs and you've already added them in your custom defs. I've refactored them to override the default ones here #9695. (You can either wait for that PR to get merged or can include the changes in that PR here and I'll close that one.)

Reply:
Done. And I've moved the changes from your PR here so that it's faster overall, thanks!